pymess/models/emails.py

```diff
@@ -248,3 +280,20 @@ class EmailTemplateDisallowedObject(BaseRelatedObject):
     class Meta(BaseRelatedObject.Meta):
         verbose_name = _('disallowed object of an e-mail template')
         verbose_name_plural = _('disallowed objects of e-mail templates')
+
+
+class StaticTemplateAttachment(SmartModel):
```

Comment:
Ten název čtu jako příloha statické šablony :) Pojmenoval bych to jako `EmailTemplateStaticAttachment`.

Reply:
Upraveno.